README.md

```diff
@@ -18,18 +18,22 @@ https://www.mongodb.com/docs/compass/master/
 
 3.Download the latest version of MongoDB Compass for Windows.
 
-## Installing Compass
+## Installing Mongo Compass
 
 1.Double-click the installer file.
 
 2.Follow the prompts to install Compass. You can select the destination of the Compass installation.
 
-3.Once installed, Compass launches and prompts you to configure privacy settings and specify update preferences.
+3.Once installed, Compass launches and prompts you to configure privacy settings and
+specify update preferences.
 
+## Installing Mongo Compass for Mac
```

Comment:
The whole section above is under the header
`build for windows`
Could we change this to 
build for mac
and copy the npm install lines into this too!